tests/baselines/reference/uniqueSymbolsDeclarations.errors.txt

```diff
@@ -0,0 +1,247 @@
+uniqueSymbolsDeclarations.ts(104,7): error TS2527: The inferred type of 'constInitToLReadonlyType' references an inaccessible 'unique symbol' type. A type annotation is necessary.
+uniqueSymbolsDeclarations.ts(105,7): error TS2527: The inferred type of 'constInitToLReadonlyNestedType' references an inaccessible 'unique symbol' type. A type annotation is necessary.

```

Comment:
These are the cases this breaks an would need to be solved differently; perhaps we can detect this through visibility instead.